shared/util/api.ts

```diff
@@ -406,7 +412,12 @@ export class API {
             }
         }
 
-        const data = await queryGraphQL<Response>(query, { query: searchQuery })
+        let vars: { [name: string]: unknown } = { query: searchQuery }
+        if (context) {
+            vars = { context, ...vars }
+        }
+
+        const data = await queryGraphQL<Response>(query, vars)
```

Comment:
This should work since if `context` is `undefined`, it'll be omitted from the JSON object
```suggestion
        const data = await queryGraphQL<Response>(query, { query: searchQuery, context })
```